File: source/Zapasovnik.APP/app/src/main/java/com/example/zapasovnik/model/TeamDetail.kt
package com.example.zapasovnik.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class TeamDetail (
    @SerialName("teamId")
    var Id: Int,

    @SerialName("teamName")
    var Name: String,

    @SerialName("teamEstablished")
    var Established: String,
)

File: source/Zapasovnik.API/DTOs/TeamDetail.cs
namespace Zapasovnik.API.DTOs
{
    public class TeamDetail
    {
        public int TeamId { get; set; }
        public string TeamName { get; set; }
        public DateTime TeamEstablished { get; set; }
        public bool isFavorite { get; set; }
    }
}
